skbio/stats/distance/_base.py

```diff
@@ -487,6 +487,36 @@ def to_data_frame(self):
         """
         return pd.DataFrame(data=self.data, index=self.ids, columns=self.ids)
 
+    @experimental(as_of="0.5.0")
```

Comment:
This should be `"0.5.0-dev"` (we change them to the right version just before release)